pkg/apiserver/endpoints/handlers/get.go

```diff
@@ -181,5 +181,5 @@ func getRequestOptions(req *http.Request, scope handlers.RequestScope, into runt
 		newQuery[subpathKey] = []string{p}
 		query = newQuery
 	}
-	return scope.ParameterCodec.DecodeParameters(query, scope.Kind.GroupVersion(), into)
```

Comment:
Personally, I like the original style, and you?
Maybe we should disable the `"-QF1008"` check from staticcheck. Like:
https://github.com/karmada-io/karmada/blob/48bab1473cc78b009c137702ceff6c1fe9a7f829/.golangci.yml#L69-L73